docker-compose-tests.yml

```diff
@@ -35,7 +35,7 @@ services:
       - gp-finder-db:es
 
   gp-finder-db:
-    image: "nhsuk/profiles-db-elastic:${profiles_db_elastic_docker_image_tag-0.2.0}"
+    image: "nhsuk/profiles-db-elastic:pr-22"
```

Comment:
This change will need updating once the profiles-db-elastic PR has been merged and a release created.